src/components/views/AddFood.vue

```diff
@@ -49,6 +50,24 @@
           class="add-food--group-children"
           @click.prevent="productGroupDialog.open(productGroup.$id)"
         >Produkt-Gruppe wählen</AtomButton>
+      </div>
+       <div class="add-food--group-container">
+        <AtomInput
+          v-model="unit.name"
+          id="add-food--group"
+          class="add-food--group-children"
+          label="Maßeinheit"
+          leading-icon="straighten"
+          required
+          no-label
+          disabled
+        />
+        <AtomButton
+          icon="list"
+          big
+          class="add-food--group-children"
+           @click.prevent="unitDialog.open(unit.$id)"
```

Comment:
Code-Styling: Fix indent
```suggestion
          @click.prevent="unitDialog.open(unit.$id)"
```